src/rgw/driver/rados/rgw_pubsub_push.cc

```diff
@@ -326,6 +337,10 @@ class RGWPubSubKafkaEndpoint : public RGWPubSubEndpoint {
       return w->wait(y);
     }
   }
+  
+  int is_alive(CephContext* cct, optional_yield y) override {
+    return kafka::check_broker(conn_name, topic);
```

Comment:
`check_broker()` is returning internal error codes like `STATUS_CONNECTION_CLOSED` that the rest of rgw doesn't understand. should there be some kind of error mapping so we can choose the right http error to return?

Reply:
changed to -EIO